cylc/flow/resources.py

```diff
@@ -61,7 +70,10 @@ def get_resources(target_dir, resources=None):
     if resources is None:
         resources = resource_names
     for resource in resources:
-        if resource not in resource_names:
+        if (
+            resource not in resource_names
+            and resource != 'etc/tutorial'
+        ):
```

Comment:
Why not just add `'etc/tutorial'` to `resource_names`?

Reply:
Because this function handles files, but not directories.

Reply:
We could make the main function support directories and make `~/cylc-src` the default dest directory for resources in the tutorial function. That way users wouldn't have to type the `--tutorial` option and the `[DIR]` positional argument would still work.

Reply:
OK, I'll do that then.